Lab2/src/UI/View/CrudConsoleService/GroupCrudConsoleService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using BLL.Models;
+using BLL.Services;
+using System;
+using UI.Interfaces;
+
+namespace UI.View.CrudConsoleService
+{
+    public class GroupCrudConsoleService : ICrudConsoleService<Group>
```

Comment:
GroupsCrudConsoleService (refactor all names for all CrudConsole services)

Reply:
Fixed